nistats/_utils/testing.py

```diff
@@ -124,14 +124,17 @@ def _create_fake_bids_dataset(base_dir='', n_sub=10, n_ses=2,
             func_path = os.path.join(subses_dir, 'func')
             os.makedirs(func_path)
             for task, n_run in zip(tasks, n_runs):
-                for run in ['run-%02d' % label for label in range(1, n_run + 1)]:
+                for run in ['run-%02d' % label
+                            for label in range(1, n_run + 1)
+                            ]:
```

Comment:
Is this really required ? Personally I don't like this (yet I agree that style is not a matter of taste)